packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -451,6 +451,11 @@ export default class BuySellStore extends BaseStore {
         this.selected_value = selected_value;
     }
 
+    @action.bound
```

Comment:
I need an action to set the `should_show_popup` to false without triggering `fetchAdvertiserAdverts()` which would re-render the adverts list in the background while the rate change error modal is visible